docker-compose.unit-tests.yml

```diff
@@ -7,11 +7,12 @@ services:
       args:
         GITHUB_TOKEN: ${GITHUB_TOKEN}
     volumes:
-      - ./artifacts/tests/docker-output/:/src/artifacts/tests/output/
+      - ./output:/output
```

Comment:
why not artifacts? we do everything via artifacts

Reply:
I can try but probably we already use the artifacts folder to build tests. I don't know why

Reply:
done